cognite_toolkit/cognite_modules/experimental/example_pump_data_model/data_models/4.Pump.view.yaml

```diff
@@ -20,7 +20,7 @@ properties:
    containerPropertyIdentifier: parent
    name: liftStation
    source:
-     external_id: LiftStation
+     externalId: LiftStation
```

Comment:
This bug does no longer pass as data model identifiers now have to use camel case.